{"id":10598,"className":"javax.management.relation.RelationException","methodName":"RelationException","signature":"\u003cjavax.management.relation.RelationException: void RelationException()\u003e","javadoc":"/** \n * Default constructor, no message put in exception.\n */\n","tf":{"freq":{"except":2,"put":1,"messag":1,"relat":1,"constructor":1},"maxFreq":2,"totalTerms":6},"loc":3}